index.html

```diff
@@ -225,20 +215,34 @@ <h2>
               </li>
               <li>
                 <p>
-                  If controller is not <code>null</code>, run the following steps:
+                  If <var>controller</var> is not <code>null</code>, run the following steps:
                 </p>
-                  <ol>
-                    <li>
-                      If <var>controller</var>.{{CaptureController/[[isBound]]}}
+                <ol>
+                  <li>
+                    <p>
+                      If <var>controller</var>.{{CaptureController/[[IsBound]]}}
                       is <code>true</code>, return a promise [=reject|rejected=] with
                       a {{DOMException}} object whose {{DOMException/name}} attribute has
                       the value {{InvalidStateError}}.
-                    </li>
-                    <li>
-                      Set <var>controller</var>.{{CaptureController/[[isBound]]}} to <code>true</code>.
-                    </li>
+                    </p>
+                  </li>
+                  <li>
+                    <p>
+                      Set <var>controller</var>.{{CaptureController/[[IsBound]]}} to <code>true</code>.
+                    </p>
+                  </li>
                 </ol>
               </li>
+              <li>
+                <p>If the [=relevant global object=] of [=this=] does not have
+          [=transient activation=], return a promise <a>rejected</a>
+                with a {{DOMException}} object whose {{DOMException/name}}
+                attribute has the value {{InvalidStateError}}.</p>
+              </li>
+              <li>
+                <p>Let <var>options</var> be the method's first
+                argument.</p>
+              </li>
```

Comment:
Sorry this isn't the part that I suggested moving. Please move it back. I meant instead we could move the controller binding steps to right before the in parallel steps. Sorry for any confusion.

Reply:
I don't recall moving this. That line 238 is mis-indented indicates something went wrong along the way. Let's see about the fix...

Reply:
Having looked at it again - I'm not following. This looks correctly placed. Please look again. (And note that it's unrelated to the request to move the focus-tracking part to the bottom.)